[gazebo] build fail due to graphviz-4.0.0

Install gazebo with `paru -S gazebo` failed.

![image](https://user-images.githubusercontent.com/43972734/173128998-5bcda7ad-d622-4084-be5a-90ff31f527cb.png)

<details>
<summary>Log</summary>
<p>

```
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/EditorView.cc: In member function ‘virtual void gazebo::gui::EditorView::wheelEvent(QWheelEvent*)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/EditorView.cc:219:23: warning: ‘QMatrix QGraphicsView::matrix() const’ is deprecated: Use transform() [-Wdeprecated-declarations]
  219 |   QMatrix mat = matrix();
      |                 ~~~~~~^~
In file included from /usr/include/qt/QtWidgets/QGraphicsView:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:54,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/Conversions.hh:24,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/EditorView.cc:25:
/usr/include/qt/QtWidgets/qgraphicsview.h:169:48: note: declared here
  169 |     QT_DEPRECATED_X("Use transform()") QMatrix matrix() const;
      |                                                ^~~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/EditorView.cc:220:38: warning: ‘QPoint QWheelEvent::pos() const’ is deprecated: Use position() [-Wdeprecated-declarations]
  220 |   QPointF mousePosition = _event->pos();
      |                           ~~~~~~~~~~~^~
In file included from /usr/include/qt/QtGui/QResizeEvent:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:34:
/usr/include/qt/QtGui/qevent.h:225:19: note: declared here
  225 |     inline QPoint pos() const { return p.toPoint(); }
      |                   ^~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/EditorView.cc:239:18: warning: ‘void QGraphicsView::setMatrix(const QMatrix&, bool)’ is deprecated: Use setTransform() [-Wdeprecated-declarations]
  239 |   this->setMatrix(mat);
      |   ~~~~~~~~~~~~~~~^~~~~
/usr/include/qt/QtWidgets/qgraphicsview.h:170:48: note: declared here
  170 |     QT_DEPRECATED_X("Use setTransform()") void setMatrix(const QMatrix &matrix, bool combine = false);
      |                                                ^~~~~~~~~
[ 69%] Automatic MOC for target ActuatorPlugin
[ 69%] Built target ActuatorPlugin_autogen
[ 69%] Automatic MOC for target AmbientOcclusionVisualPlugin
[ 69%] Built target AmbientOcclusionVisualPlugin_autogen
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/GrabberHandle.cc.o
In file included from /usr/include/OGRE/OgreMeshSerializerImpl.h:37,
                 from /usr/include/OGRE/OgreMeshSerializer.h:34,
                 from /usr/include/OGRE/Ogre.h:75,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/rendering/ogre_gazebo.h:26,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/rendering/Conversions.hh:25,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/rendering/Camera.hh:47,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/build/gazebo/rendering/rendering.hh:5,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/test/ServerFixture.hh:53,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/test/ServerFixture.cc:26:
/usr/include/OGRE/OgreEdgeListBuilder.h: In member function ‘bool Ogre::EdgeListBuilder::geometryLess::operator()(const Ogre::EdgeListBuilder::Geometry&, const Ogre::EdgeListBuilder::Geometry&) const’:
/usr/include/OGRE/OgreEdgeListBuilder.h:227: note: ‘-Wmisleading-indentation’ is disabled from this point onwards, since column-tracking was disabled due to the size of the code/headers
  227 |                 if (a.vertexSet < b.vertexSet) return true;
      | 
/usr/include/OGRE/OgreEdgeListBuilder.h:227: note: adding ‘-flarge-source-files’ will allow for more column-tracking support, at the expense of compilation time and memory
In file included from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/sensors/SensorTypes.hh:23,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/sensors/Sensor.hh:31,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/sensors/AltimeterSensor.hh:25,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/build/gazebo/sensors/sensors.hh:2,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/test/ServerFixture.hh:52,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/test/ServerFixtureRecord.cc:21:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/common/EnumIface.hh:180:12: warning: ‘template<class _Category, class _Tp, class _Distance, class _Pointer, class _Reference> struct std::iterator’ is deprecated [-Wdeprecated-declarations]
  180 |     : std::iterator<std::bidirectional_iterator_tag, Enum>
      |            ^~~~~~~~
In file included from /usr/include/c++/12.1.0/string:45,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/test/ServerFixtureRecord.cc:17:
/usr/include/c++/12.1.0/bits/stl_iterator_base_types.h:127:34: note: declared here
  127 |     struct _GLIBCXX17_DEPRECATED iterator
      |                                  ^~~~~~~~
[ 69%] Linking CXX executable gzserver
In file included from /usr/include/ignition/msgs5/ignition/msgs/MessageTypes.hh:119,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/rendering/Visual.hh:32,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/BuildingMaker.cc:36:
/usr/include/ignition/msgs5/ignition/msgs/model_configuration.pb.h: In member function ‘ignition::msgs::ModelConfiguration& ignition::msgs::ModelConfiguration::operator=(ignition::msgs::ModelConfiguration&&)’:
/usr/include/ignition/msgs5/ignition/msgs/model_configuration.pb.h:94: note: ‘-Wmisleading-indentation’ is disabled from this point onwards, since column-tracking was disabled due to the size of the code/headers
   94 |     if (this == &from) return *this;
      | 
/usr/include/ignition/msgs5/ignition/msgs/model_configuration.pb.h:94: note: adding ‘-flarge-source-files’ will allow for more column-tracking support, at the expense of compilation time and memory
[ 69%] Built target gzserver
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/GridLines.cc.o
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/ImportImageDialog.cc.o
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/ImportImageView.cc.o
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/LevelInspectorDialog.cc.o
[ 69%] Automatic MOC for target ArduCopterPlugin
[ 69%] Built target ArduCopterPlugin_autogen
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/LevelWidget.cc.o
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/MeasureItem.cc.o
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/RectItem.cc.o
In file included from /usr/include/OGRE/Terrain/OgreTerrainMaterialGeneratorA.h:33,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/rendering/ogre_gazebo.h:62,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/rendering/Conversions.hh:25,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/rendering/Camera.hh:47,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/build/gazebo/rendering/rendering.hh:5,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/test/ServerFixture.hh:53:
/usr/include/OGRE/Terrain/OgreTerrainMaterialGenerator.h: In member function ‘Ogre::TerrainMaterialGenerator::Profile* Ogre::TerrainMaterialGenerator::getActiveProfile() const’:
/usr/include/OGRE/Terrain/OgreTerrainMaterialGenerator.h:248: note: ‘-Wmisleading-indentation’ is disabled from this point onwards, since column-tracking was disabled due to the size of the code/headers
  248 |                         if (!mActiveProfile && !mProfiles.empty())
      | 
/usr/include/OGRE/Terrain/OgreTerrainMaterialGenerator.h:248: note: adding ‘-flarge-source-files’ will allow for more column-tracking support, at the expense of compilation time and memory
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/MeasureItem.cc: In member function ‘virtual void gazebo::gui::MeasureItem::paint(QPainter*, const QStyleOptionGraphicsItem*, QWidget*)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/MeasureItem.cc:87:50: warning: ‘int QFontMetrics::width(const QString&, int) const’ is deprecated: Use QFontMetrics::horizontalAdvance [-Wdeprecated-declarations]
   87 |   float textWidth = _painter->fontMetrics().width(stream.str().c_str());
      |                     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/qt/QtGui/qpainter.h:59,
                 from /usr/include/qt/QtGui/QPainter:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:33,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/MeasureItem.hh:24,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/building/MeasureItem.cc:23:
/usr/include/qt/QtGui/qfontmetrics.h:106:9: note: declared here
  106 |     int width(const QString &, int len = -1) const;
      |         ^~~~~
[ 69%] Automatic MOC for target ArrangePlugin
[ 69%] Built target ArrangePlugin_autogen
[ 69%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/RotateHandle.cc.o
[ 71%] Automatic MOC for target AttachLightPlugin
[ 71%] Built target AttachLightPlugin_autogen
[ 71%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/ScaleWidget.cc.o
[ 71%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/SegmentItem.cc.o
[ 71%] Automatic MOC for target BlinkVisualPlugin
[ 71%] Built target BlinkVisualPlugin_autogen
[ 71%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/StairsInspectorDialog.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/StairsItem.cc.o
[ 73%] Automatic MOC for target BreakableJointPlugin
[ 73%] Built target BreakableJointPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/WallInspectorDialog.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/WallSegmentItem.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/WindowDoorInspectorDialog.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/building/WindowItem.cc.o
[ 73%] Linking CXX static library libgazebo_test_fixture.a
[ 73%] Built target gazebo_test_fixture
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/CollisionConfig.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/EditorMaterialSwitcher.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ExtrudeDialog.cc.o
[ 73%] Automatic MOC for target BuoyancyPlugin
[ 73%] Built target BuoyancyPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ImportDialog.cc.o
[ 73%] Automatic MOC for target CameraPlugin
[ 73%] Built target CameraPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/JointCreationDialog.cc.o
[ 73%] Automatic MOC for target CartDemoPlugin
[ 73%] Built target CartDemoPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/JointInspector.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/JointMaker.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/LinkConfig.cc.o
[ 73%] Automatic MOC for target CessnaPlugin
[ 73%] Built target CessnaPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/LinkInspector.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/MEUserCmdManager.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ModelCreator.cc.o
[ 73%] Automatic MOC for target ContactPlugin
[ 73%] Built target ContactPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ModelData.cc.o
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/EditorMaterialSwitcher.cc: In member function ‘void gazebo::gui::EditorMaterialSwitcher::SetMaterialScheme(const std::string&)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/EditorMaterialSwitcher.cc:54: note: ‘-Wmisleading-indentation’ is disabled from this point onwards, since column-tracking was disabled due to the size of the code/headers
   54 |   if (!this->camera || !this->camera->OgreViewport())
      | 
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/EditorMaterialSwitcher.cc:54: note: adding ‘-flarge-source-files’ will allow for more column-tracking support, at the expense of compilation time and memory
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ModelEditor.cc.o
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ModelEditorEvents.cc.o
In file included from /usr/include/boost/multi_index/detail/ord_index_impl.hpp:65,
                 from /usr/include/boost/multi_index/ordered_index.hpp:17,
                 from /usr/include/boost/property_tree/ptree.hpp:24,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/GuiIface.hh:21,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/JointMaker.cc:33:
/usr/include/boost/multi_index/detail/ord_index_node.hpp: In static member function ‘static void boost::multi_index::detail::ordered_index_node_impl<AugmentPolicy, Allocator>::decrement(pointer&)’:
/usr/include/boost/multi_index/detail/ord_index_node.hpp:289: note: ‘-Wmisleading-indentation’ is disabled from this point onwards, since column-tracking was disabled due to the size of the code/headers
  289 |       while(y->right()!=pointer(0))y=y->right();
      | 
/usr/include/boost/multi_index/detail/ord_index_node.hpp:289: note: adding ‘-flarge-source-files’ will allow for more column-tracking support, at the expense of compilation time and memory
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ModelEditorPalette.cc.o
[ 73%] Automatic MOC for target ContainPlugin
[ 73%] Built target ContainPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ModelPluginInspector.cc.o
[ 73%] Automatic MOC for target DepthCameraPlugin
[ 73%] Built target DepthCameraPlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/ModelTreeWidget.cc.o
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/ModelEditorPalette.cc: In constructor ‘gazebo::gui::ModelEditorPalette::ModelEditorPalette(QWidget*)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/ModelEditorPalette.cc:130:26: warning: ‘constexpr QFlags<T>::QFlags(Zero) [with Enum = Qt::AlignmentFlag; Zero = int QFlags<Qt::AlignmentFlag>::Private::*]’ is deprecated: Use default constructor instead [-Wdeprecated-declarations]
  130 |   customLayout->addWidget(customButton, 0, 0);
      |   ~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~
In file included from /usr/include/qt/QtCore/qglobal.h:1299,
                 from /usr/include/qt/QtCore/QtCore:4,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:25,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/Actions.hh:22,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/ModelEditorPalette.cc:22:
/usr/include/qt/QtCore/qflags.h:123:80: note: declared here
  123 |     QT_DEPRECATED_X("Use default constructor instead") Q_DECL_CONSTEXPR inline QFlags(Zero) noexcept : i(0) {}
      |                                                                                ^~~~~~
[ 73%] Automatic MOC for target DiffDrivePlugin
[ 73%] Built target DiffDrivePlugin_autogen
[ 73%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/VisualConfig.cc.o
In file included from /usr/include/qt/QtCore/qobject.h:49,
                 from /usr/include/qt/QtCore/qabstractanimation.h:43,
                 from /usr/include/qt/QtCore/QtCore:6,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:25,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/Actions.hh:22,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/ModelCreator.cc:40:
/usr/include/qt/QtCore/qlist.h: In member function ‘void QList<T>::setSharable(bool)’:
/usr/include/qt/QtCore/qlist.h:191: note: ‘-Wmisleading-indentation’ is disabled from this point onwards, since column-tracking was disabled due to the size of the code/headers
  191 |         if (sharable == d->ref.isSharable())
      | 
/usr/include/qt/QtCore/qlist.h:191: note: adding ‘-flarge-source-files’ will allow for more column-tracking support, at the expense of compilation time and memory
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/GraphScene.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/GraphView.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/model/SchematicViewWidget.cc.o
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/GraphView.cc: In member function ‘virtual void gazebo::gui::GraphView::wheelEvent(QWheelEvent*)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/GraphView.cc:77:23: warning: ‘QMatrix QGraphicsView::matrix() const’ is deprecated: Use transform() [-Wdeprecated-declarations]
   77 |   QMatrix mat = matrix();
      |                 ~~~~~~^~
In file included from /usr/include/qt/QtWidgets/QGraphicsView:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:54,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/GraphView.hh:20,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/GraphView.cc:22:
/usr/include/qt/QtWidgets/qgraphicsview.h:169:48: note: declared here
  169 |     QT_DEPRECATED_X("Use transform()") QMatrix matrix() const;
      |                                                ^~~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/GraphView.cc:78:38: warning: ‘QPoint QWheelEvent::pos() const’ is deprecated: Use position() [-Wdeprecated-declarations]
   78 |   QPointF mousePosition = _event->pos();
      |                           ~~~~~~~~~~~^~
In file included from /usr/include/qt/QtGui/QResizeEvent:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:34:
/usr/include/qt/QtGui/qevent.h:225:19: note: declared here
  225 |     inline QPoint pos() const { return p.toPoint(); }
      |                   ^~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/model/GraphView.cc:97:18: warning: ‘void QGraphicsView::setMatrix(const QMatrix&, bool)’ is deprecated: Use setTransform() [-Wdeprecated-declarations]
   97 |   this->setMatrix(mat);
      |   ~~~~~~~~~~~~~~~^~~~~
/usr/include/qt/QtWidgets/qgraphicsview.h:170:48: note: declared here
  170 |     QT_DEPRECATED_X("Use setTransform()") void setMatrix(const QMatrix &matrix, bool combine = false);
      |                                                ^~~~~~~~~
[ 76%] Automatic MOC for target FiducialCameraPlugin
[ 76%] Built target FiducialCameraPlugin_autogen
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/EditableLabel.cc.o
[ 76%] Automatic MOC for target FlashLightPlugin
[ 76%] Built target FlashLightPlugin_autogen
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/ExportDialog.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/IncrementalPlot.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/IntrospectionCurveHandler.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/Palette.cc.o
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc: In member function ‘virtual void PlotViewDelegate::paint(QPainter*, const QStyleOptionViewItem&, const QModelIndex&) const’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc:104:43: warning: ‘int QFontMetrics::width(const QString&, int) const’ is deprecated: Use QFontMetrics::horizontalAdvance [-Wdeprecated-declarations]
  104 |             int checkTitleWidth = fm.width(title) + checkSize + checkMargin;
      |                                   ~~~~~~~~^~~~~~~
In file included from /usr/include/qt/QtGui/qpainter.h:59,
                 from /usr/include/qt/QtGui/QPainter:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:33,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.hh:23,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc:20:
/usr/include/qt/QtGui/qfontmetrics.h:106:9: note: declared here
  106 |     int width(const QString &, int len = -1) const;
      |         ^~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc: In constructor ‘gazebo::gui::ExportDialog::ExportDialog(QWidget*, const std::__cxx11::list<gazebo::gui::PlotCanvas*>&)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc:239:35: warning: ‘static QPixmap QPixmap::grabWindow(WId, int, int, int, int)’ is deprecated: Use QScreen::grabWindow() instead [-Wdeprecated-declarations]
  239 |     QIcon icon(QPixmap::grabWindow(plot->winId()));
      |                ~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~
In file included from /usr/include/qt/QtGui/qbitmap.h:44,
                 from /usr/include/qt/QtGui/QBitmap:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:31:
/usr/include/qt/QtGui/qpixmap.h:118:20: note: declared here
  118 |     static QPixmap grabWindow(WId, int x = 0, int y = 0, int w = -1, int h = -1);
      |                    ^~~~~~~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc: In member function ‘void gazebo::gui::ExportDialog::OnExport(gazebo::gui::FileType)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc:315:39: warning: ‘QFileDialog::DirectoryOnly’ is deprecated: Use setOption(ShowDirsOnly, true) instead [-Wdeprecated-declarations]
  315 |   fileDialog.setFileMode(QFileDialog::DirectoryOnly);
      |                                       ^~~~~~~~~~~~~
In file included from /usr/include/qt/QtWidgets/QFileDialog:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:48:
/usr/include/qt/QtWidgets/qfiledialog.h:84:21: note: declared here
   84 |                     DirectoryOnly Q_DECL_ENUMERATOR_DEPRECATED_X("Use setOption(ShowDirsOnly, true) instead")};
      |                     ^~~~~~~~~~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/ExportDialog.cc:315:39: warning: ‘QFileDialog::DirectoryOnly’ is deprecated: Use setOption(ShowDirsOnly, true) instead [-Wdeprecated-declarations]
  315 |   fileDialog.setFileMode(QFileDialog::DirectoryOnly);
      |                                       ^~~~~~~~~~~~~
/usr/include/qt/QtWidgets/qfiledialog.h:84:21: note: declared here
   84 |                     DirectoryOnly Q_DECL_ENUMERATOR_DEPRECATED_X("Use setOption(ShowDirsOnly, true) instead")};
      |                     ^~~~~~~~~~~~~
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/PlotCanvas.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/PlotCurve.cc.o
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/IncrementalPlot.cc: In member function ‘virtual void gazebo::gui::PlotMagnifier::widgetWheelEvent(QWheelEvent*)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/IncrementalPlot.cc:58:50: warning: ‘QPoint QWheelEvent::pos() const’ is deprecated: Use position() [-Wdeprecated-declarations]
   58 |                 this->mousePos = _wheelEvent->pos();
      |                                  ~~~~~~~~~~~~~~~~^~
In file included from /usr/include/qt/QtGui/QResizeEvent:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:34,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/PlotCurve.hh:26,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/IncrementalPlot.cc:25:
/usr/include/qt/QtGui/qevent.h:225:19: note: declared here
  225 |     inline QPoint pos() const { return p.toPoint(); }
      |                   ^~~
[ 76%] Automatic MOC for target FollowerPlugin
[ 76%] Built target FollowerPlugin_autogen
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/PlotManager.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/PlotTracker.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/PlotWindow.cc.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/TopicCurveHandler.cc.o
[ 76%] Automatic MOC for target ForceTorquePlugin
[ 76%] Built target ForceTorquePlugin_autogen
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/VariablePill.cc.o
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/PlotTracker.cc: In member function ‘QString gazebo::gui::PlotTracker::CurveInfoAt(const QwtPlotCurve*, const QPointF&) const’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/PlotTracker.cc:242:21: warning: ‘QString::null’ is deprecated: use QString() [-Wdeprecated-declarations]
  242 |     return QString::null;
      |                     ^~~~
In file included from /usr/include/qt/QtCore/qobject.h:47,
                 from /usr/include/qt/QtCore/qabstractanimation.h:43,
                 from /usr/include/qt/QtCore/QtCore:6,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:25,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/PlotTracker.cc:20:
/usr/include/qt/QtCore/qstring.h:954:23: note: declared here
  954 |     static const Null null;
      |                       ^~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/PlotTracker.cc:242:21: warning: ‘QString::null’ is deprecated: use QString() [-Wdeprecated-declarations]
  242 |     return QString::null;
      |                     ^~~~
/usr/include/qt/QtCore/qstring.h:954:23: note: declared here
  954 |     static const Null null;
      |                       ^~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/Palette.cc: In member function ‘virtual void PlotItemDelegate::paint(QPainter*, const QStyleOptionViewItem&, const QModelIndex&) const’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/Palette.cc:237:40: warning: ‘int QFontMetrics::width(const QString&, int) const’ is deprecated: Use QFontMetrics::horizontalAdvance [-Wdeprecated-declarations]
  237 |         textRect.adjust(fmRegular.width(textStr), 0, 0, 0);
      |                         ~~~~~~~~~~~~~~~^~~~~~~~~
In file included from /usr/include/qt/QtGui/qpainter.h:59,
                 from /usr/include/qt/QtGui/QPainter:1,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:33,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/ConfigWidget.hh:30,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/Palette.cc:30:
/usr/include/qt/QtGui/qfontmetrics.h:106:9: note: declared here
  106 |     int width(const QString &, int len = -1) const;
      |         ^~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/Palette.cc:247:37: warning: ‘int QFontMetrics::width(const QString&, int) const’ is deprecated: Use QFontMetrics::horizontalAdvance [-Wdeprecated-declarations]
  247 |         textRect.adjust(fmBold.width(textStr), 0, 0, 0);
      |                         ~~~~~~~~~~~~^~~~~~~~~
/usr/include/qt/QtGui/qfontmetrics.h:106:9: note: declared here
  106 |     int width(const QString &, int len = -1) const;
      |         ^~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/Palette.cc: In member function ‘void gazebo::gui::SearchModel::SetSearch(const QString&)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/plot/Palette.cc:439:22: warning: ‘void QSortFilterProxyModel::filterChanged()’ is deprecated: Use QSortFilterProxyModel::invalidateFilter [-Wdeprecated-declarations]
  439 |   this->filterChanged();
      |   ~~~~~~~~~~~~~~~~~~~^~
In file included from /usr/include/qt/QtCore/QtCore:214,
                 from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qt.h:25:
/usr/include/qt/QtCore/qsortfilterproxymodel.h:143:73: note: declared here
  143 |     QT_DEPRECATED_X("Use QSortFilterProxyModel::invalidateFilter") void filterChanged();
      |                                                                         ^~~~~~~~~~~~~
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/plot/VariablePillContainer.cc.o
[ 76%] Automatic MOC for target GimbalSmall2dPlugin
[ 76%] Built target GimbalSmall2dPlugin_autogen
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVCore.cpp.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVGraphPrivate.cpp.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVEdgePrivate.cpp.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVGvcPrivate.cpp.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVNodePrivate.cpp.o
[ 76%] Building CXX object gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/QGVEdge.cpp.o
In file included from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/private/QGVCore.cpp:19:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/private/QGVCore.h: In static member function ‘static Agraph_t* QGVCore::agmemread2(const char*)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/private/QGVCore.h:99:17: error: ‘Agiodisc_t’ {aka ‘struct Agiodisc_s’} has no member named ‘putstr’
   99 |       memIoDisc.putstr = AgIoDisc.putstr;
      |                 ^~~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/private/QGVCore.h:99:35: error: ‘Agiodisc_t’ {aka ‘struct Agiodisc_s’} has no member named ‘putstr’
   99 |       memIoDisc.putstr = AgIoDisc.putstr;
      |                                   ^~~~~~
make[2]: *** [gazebo/gui/CMakeFiles/gazebo_gui.dir/build.make:1734: gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/private/QGVCore.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 76%] Automatic MOC for target GpuRayPlugin
[ 76%] Built target GpuRayPlugin_autogen
[ 76%] Automatic MOC for target HarnessPlugin
[ 76%] Built target HarnessPlugin_autogen
[ 76%] Automatic MOC for target HeightmapLODPlugin
[ 76%] Built target HeightmapLODPlugin_autogen
[ 76%] Automatic MOC for target ImuSensorPlugin
[ 76%] Built target ImuSensorPlugin_autogen
[ 76%] Automatic MOC for target InitialVelocityPlugin
[ 76%] Built target InitialVelocityPlugin_autogen
[ 76%] Automatic MOC for target JointControlPlugin
[ 76%] Built target JointControlPlugin_autogen
[ 76%] Automatic MOC for target JointTrajectoryPlugin
[ 76%] Built target JointTrajectoryPlugin_autogen
[ 76%] Automatic MOC for target KeysToCmdVelPlugin
[ 76%] Built target KeysToCmdVelPlugin_autogen
[ 76%] Automatic MOC for target KeysToJointsPlugin
[ 76%] Built target KeysToJointsPlugin_autogen
[ 76%] Automatic MOC for target LensFlareSensorPlugin
[ 76%] Built target LensFlareSensorPlugin_autogen
[ 76%] Automatic MOC for target LiftDragPlugin
[ 76%] Built target LiftDragPlugin_autogen
[ 76%] Automatic MOC for target LinearBatteryConsumerPlugin
[ 76%] Built target LinearBatteryConsumerPlugin_autogen
[ 76%] Automatic MOC for target LinearBatteryPlugin
[ 76%] Built target LinearBatteryPlugin_autogen
[ 76%] Automatic MOC for target LinkPlot3DPlugin
In file included from /home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/QGVEdge.cpp:20:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/private/QGVCore.h: In static member function ‘static Agraph_t* QGVCore::agmemread2(const char*)’:
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/private/QGVCore.h:99:17: error: ‘Agiodisc_t’ {aka ‘struct Agiodisc_s’} has no member named ‘putstr’
   99 |       memIoDisc.putstr = AgIoDisc.putstr;
      |                 ^~~~~~
/home/dh/.cache/paru/clone/gazebo/src/gazebo-gazebo11_11.10.2/gazebo/gui/qgv/private/QGVCore.h:99:35: error: ‘Agiodisc_t’ {aka ‘struct Agiodisc_s’} has no member named ‘putstr’
   99 |       memIoDisc.putstr = AgIoDisc.putstr;
      |                                   ^~~~~~
[ 76%] Built target LinkPlot3DPlugin_autogen
[ 76%] Automatic MOC for target MisalignmentPlugin
[ 76%] Built target MisalignmentPlugin_autogen
[ 76%] Automatic MOC for target ModelPropShop
[ 76%] Built target ModelPropShop_autogen
[ 76%] Automatic MOC for target MudPlugin
[ 76%] Built target MudPlugin_autogen
[ 76%] Automatic MOC for target PlaneDemoPlugin
[ 76%] Built target PlaneDemoPlugin_autogen
[ 76%] Automatic MOC for target PressurePlugin
[ 76%] Built target PressurePlugin_autogen
[ 76%] Automatic MOC for target RayPlugin
make[2]: *** [gazebo/gui/CMakeFiles/gazebo_gui.dir/build.make:1804: gazebo/gui/CMakeFiles/gazebo_gui.dir/qgv/QGVEdge.cpp.o] Error 1
[ 76%] Built target RayPlugin_autogen
[ 76%] Automatic MOC for target RaySensorNoisePlugin
[ 76%] Automatic MOC for target ReflectancePlugin
[ 76%] Built target RaySensorNoisePlugin_autogen
[ 76%] Built target ReflectancePlugin_autogen
[ 76%] Automatic MOC for target RubblePlugin
[ 78%] Automatic MOC for target ShaderParamVisualPlugin
[ 78%] Built target RubblePlugin_autogen
[ 78%] Automatic MOC for target SkidSteerDrivePlugin
[ 78%] Built target ShaderParamVisualPlugin_autogen
[ 78%] Automatic MOC for target SonarPlugin
[ 78%] Built target SkidSteerDrivePlugin_autogen
[ 78%] Automatic MOC for target SphereAtlasDemoPlugin
[ 78%] Built target SonarPlugin_autogen
[ 78%] Automatic MOC for target StaticMapPlugin
[ 78%] Built target SphereAtlasDemoPlugin_autogen
[ 78%] Automatic MOC for target StopWorldPlugin
[ 78%] Built target StaticMapPlugin_autogen
[ 78%] Built target StopWorldPlugin_autogen
[ 78%] Automatic MOC for target TouchPlugin
[ 78%] Automatic MOC for target VariableGearboxPlugin
[ 78%] Built target TouchPlugin_autogen
[ 78%] Automatic MOC for target VehiclePlugin
[ 78%] Built target VariableGearboxPlugin_autogen
[ 78%] Automatic MOC for target WheelSlipPlugin
[ 78%] Built target VehiclePlugin_autogen
[ 78%] Built target WheelSlipPlugin_autogen
[ 78%] Automatic MOC for target WindPlugin
[ 78%] Automatic MOC for target HydraPlugin
[ 78%] Built target WindPlugin_autogen
[ 78%] Built target HydraPlugin_autogen
[ 78%] Automatic MOC for target HydraDemoPlugin
[ 78%] Automatic MOC for target JoyPlugin
[ 78%] Built target HydraDemoPlugin_autogen
[ 78%] Built target JoyPlugin_autogen
[ 78%] Automatic MOC for target ElevatorPlugin
[ 78%] Automatic MOC for target RandomVelocityPlugin
[ 78%] Built target ElevatorPlugin_autogen
[ 78%] Automatic MOC for target TransporterPlugin
[ 78%] Built target RandomVelocityPlugin_autogen
[ 78%] Building CXX object plugins/CMakeFiles/TrackedVehiclePlugin.dir/TrackedVehiclePlugin_autogen/mocs_compilation.cpp.o
[ 78%] Built target TransporterPlugin_autogen
[ 78%] Building CXX object plugins/CMakeFiles/TrackedVehiclePlugin.dir/TrackedVehiclePlugin.cc.o
[ 78%] Building CXX object plugins/CMakeFiles/ActorPlugin.dir/ActorPlugin_autogen/mocs_compilation.cpp.o
[ 78%] Building CXX object plugins/CMakeFiles/ActorPlugin.dir/ActorPlugin.cc.o
[ 78%] Building CXX object plugins/CMakeFiles/ActuatorPlugin.dir/ActuatorPlugin_autogen/mocs_compilation.cpp.o
[ 78%] Building CXX object plugins/CMakeFiles/ActuatorPlugin.dir/ActuatorPlugin.cc.o
[ 78%] Building CXX object plugins/CMakeFiles/AmbientOcclusionVisualPlugin.dir/AmbientOcclusionVisualPlugin_autogen/mocs_compilation.cpp.o
[ 78%] Building CXX object plugins/CMakeFiles/AmbientOcclusionVisualPlugin.dir/AmbientOcclusionVisualPlugin.cc.o
[ 78%] Building CXX object plugins/CMakeFiles/ArrangePlugin.dir/ArrangePlugin_autogen/mocs_compilation.cpp.o
[ 78%] Building CXX object plugins/CMakeFiles/ArduCopterPlugin.dir/ArduCopterPlugin_autogen/mocs_compilation.cpp.o
[ 78%] Building CXX object plugins/CMakeFiles/ArrangePlugin.dir/ArrangePlugin.cc.o
[ 78%] Building CXX object plugins/CMakeFiles/ArduCopterPlugin.dir/ArduCopterPlugin.cc.o
[ 78%] Building CXX object plugins/CMakeFiles/AttachLightPlugin.dir/AttachLightPlugin_autogen/mocs_compilation.cpp.o
[ 78%] Building CXX object plugins/CMakeFiles/AttachLightPlugin.dir/AttachLightPlugin.cc.o
make[1]: *** [CMakeFiles/Makefile2:6135: gazebo/gui/CMakeFiles/gazebo_gui.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 78%] Linking CXX shared library libActorPlugin.so
[ 78%] Built target ActorPlugin
[ 78%] Linking CXX shared library libActuatorPlugin.so
[ 78%] Built target ActuatorPlugin
[ 78%] Linking CXX shared library libTrackedVehiclePlugin.so
[ 78%] Built target TrackedVehiclePlugin
[ 78%] Linking CXX shared library libAmbientOcclusionVisualPlugin.so
[ 78%] Linking CXX shared library libArrangePlugin.so
[ 78%] Built target AmbientOcclusionVisualPlugin
[ 78%] Built target ArrangePlugin
[ 78%] Linking CXX shared library libArduCopterPlugin.so
[ 78%] Built target ArduCopterPlugin
[ 78%] Linking CXX shared library libAttachLightPlugin.so
[ 78%] Built target AttachLightPlugin
make: *** [Makefile:146: all] Error 2
==> ERROR: A failure occurred in build().
    Aborting...
error: failed to build 'gazebo-11.10.2-3': 
error: packages failed to build: gazebo-11.10.2-3
⏎                                                                                                                  
```
</p></details>

--- a/gazebo/.SRCINFO
+++ b/gazebo/.SRCINFO
@@ -1,7 +1,7 @@
 pkgbase = gazebo
 	pkgdesc = A multi-robot simulator for outdoor environments
 	pkgver = 11.11.0
-	pkgrel = 1
+	pkgrel = 2
 	url = http://gazebosim.org/
 	install = gazebo.install
 	arch = i686

--- a/gazebo/PKGBUILD
+++ b/gazebo/PKGBUILD
@@ -9,7 +9,7 @@
 
 pkgname=gazebo
 pkgver=11.11.0
-pkgrel=1
+pkgrel=2
 pkgdesc="A multi-robot simulator for outdoor environments"
 arch=('i686' 'x86_64')
 url="http://gazebosim.org/"
@@ -30,12 +30,6 @@ makedepends=('cmake' 'doxygen' 'ruby-ronn')
 install="${pkgname}.install"
 source=("${pkgname}-${pkgver}.tar.gz::https://github.com/osrf/gazebo/archive/${pkgname}11_$pkgver.tar.gz")
 sha256sums=('e6a1965198378a8360ab7fce465990f11951629a833e518c2163c645d015354a')
-
-prepare() {
-  cd "${srcdir}/${pkgname}-${pkgname}11_$pkgver"
-
-  sed -i 's/putstr/printf/g' gazebo/gui/qgv/private/QGVCore.h
-}
 
 build() {
   cd "${srcdir}/${pkgname}-${pkgname}11_$pkgver"

